Inputs/PCATest.yaml

```diff
@@ -20,9 +20,9 @@ Systematics:
     DetID: '1'
     Error: '0.059243759525005685'
     KinematicCuts:
-      -Horncurrent: '1'
-      -TrueNeutrinoEnergy: [0.0, 0.4]
-      NeutrinoFlavourUnosc: 14
+      - Horncurrent: 1
+      - TrueNeutrinoEnergy: [0.0, 0.4]
+    NeutrinoFlavourUnosc: 14
```

Comment:
Oh crap... good spot
I think this will not work
`- Horncurrent: 1 -> - Horncurrent: [0.5, 1.5]`
and I think NeutrinoFlavourUnosc should be vector (I don't rember why)
`NeutrinoFlavourUnosc: 14 -> NeutrinoFlavourUnosc: [14]`

Reply:
Ahh that's why there were exceptions thrown from YAML about a `vector<int>`. That makes sense now!

Done these - ready to merge now